content/api/commands/intercept.md

```diff
@@ -584,6 +584,34 @@ cy.intercept('/users', (req) => {
 })
 ```
 
+#### Throttle or delay response all incoming responses
+
+You can throttle or delay all incoming responses using a `beforeEach()` in the `cypress/support/index.js` file
+
+```ts
+// Code from Real World App (RWA)
+// cypress/support/index.ts
+import { isMobile } from './utils'
+import './commands'
+// Throttle API responses for mobile testing to simulate real world conditions
```

Comment:
it feels confusing to reference these imports from RWA in the example section of these docs, idk, is this necessary/is there a better way to present this?

Reply:
Sure, but we have them elsewhere in the guide and want to direct folks from the docs to the RWA for a runnable example.  @amirrustam Might have more thoughts.

Reply:
Idk why this was marked as resolved and merged with no comment, I think this needs to be revisited still. This is a pretty clunky way to introduce RWA imo.

Reply:
@flotwig I do agree, I have it on the list to discuss and improve in our docs improvement meeting today. I think this was just intermediary add for the use case in time for the Monday release.

Overall, we want some better context for users when we talk about the RWA.